my-app/components/personal_info/personal.module.scss

```diff
@@ -0,0 +1,42 @@
+.main{
+  margin: 1rem;
+  width: 25%;
+  padding: 1rem;
+  box-shadow: rgba(149, 157, 165, 0.2) 0 8px 24px;
+  font-size: 16px;
+  line-height: 7mm;
+  p{
+    margin: 0;
+  }
+  .header{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 2rem;
+    font-weight: bold;
+    font-size: 13px;
+    p:first-child{
+      color:#00ffcd ;
+      font-size: 15px;
+      span{
+        font-weight: normal;
+      }
+    }
+    span{
+      color: grey;
+    }
+    div:nth-child(2){
+      display: flex;
+      align-items: center;
+    }
+  }
+  .details{
+    font-size: 14px;
+    .wrapper{
```

Comment:
All the content with this wrapper class is not aligned properly. Have a look on this one.